.github/workflows/test_api_and_frontend.yaml

```diff
@@ -189,3 +189,10 @@ jobs:
         with:
           name: test-results
           path: test-results
+
+      - name: Upload generated plots
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: generated-plots
+          path: skyportal/tests/data/*_expected.png
```

Comment:
Save the generated plots for us to debug (these are tiny png's so should not be an issue storage wise)